x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/security/xcontent/XContentUtilsTests.java

```diff
@@ -67,6 +71,51 @@ public void testAddAuthorizationInfoWithServiceAccount() throws IOException {
         assertThat(json, equalTo("{\"authorization\":{\"service_account\":\"" + account + "\"}}"));
     }
 
+    public void testAddAuthorizationInfoWithCrossClusterAccess() throws IOException {
+        final CrossClusterAccessSubjectInfo crossClusterAccessSubjectInfo = AuthenticationTestHelper.randomCrossClusterAccessSubjectInfo(
+            AuthenticationTestHelper.builder().build()
+        );
+        assert false == crossClusterAccessSubjectInfo.getAuthentication().isCrossClusterAccess();
+        final Authentication authentication = AuthenticationTestHelper.builder()
```

Comment:
Nit: I think we can just user `builder.crossCluster().build()` instead, i.e.:

```java
final Authentication authentication = AuthenticationTestHelper.builder().crossClusterAccess().build();
final var apiKeyName = (String) authentication.getAuthenticatingSubject().getMetadata().get(API_KEY_NAME_KEY);
final var innerAuthentication = (Authentication) authentication.getAuthenticatingSubject()
    .getMetadata()
    .get(CROSS_CLUSTER_ACCESS_AUTHENTICATION_KEY);
```

Reply:
Thanks I forgot we have this convenient method.